static/js/apps/eval_rig/util.ts

```diff
@@ -40,4 +40,11 @@ export const processText = (text: string): string => {
       return `<span class="annotation annotation-${callId}">${innerHtml}</span>`;
     }
   );
+  // Format the links
+  const httpPattern = /https:\/\/[^\s]+/g;
```

Comment:
nit: can make this a global constant outside the function?

Reply:
Done